go-controller/pkg/ovn/obj_retry.go

```diff
@@ -973,7 +1193,7 @@ func (oc *Controller) getSyncResourcesFunc(r *retryObjs) (func([]interface{}) er
 			// This needs to be a write lock because there's no locking around 'gress policies
 			extraParameters.np.Lock()
 			defer extraParameters.np.Unlock()
-			// We load the existing address set into the 'gress policy.
+			// We load the existing address set into the 'egress policy.
```

Comment:
I suppose the apostrophe somehow was a shortcut for egress and ingress :)

Reply:
ingress policy ? there is no such thing AFAIK ?